app/assets/javascripts/votes.js

```diff
@@ -0,0 +1,73 @@
+// $('#small').id = "big";
+// $('#big').id = "small";
+// $('#small').append("hi");
+
+$(document).ready(function() {
+  $(".voting").on("click", 'span', function(event) {
+    event.preventDefault();
+    var userId = $('.username').attr('id')
+    var this_vote_container = this.parentElement;
+    var reviewId = $(this_vote_container).attr('review_id')
+    var request = $.ajax({
+      url: ("/votes"),
+      type: "POST",
+      data: {
+        vote: {
+          user_id: userId,
+          review_id: reviewId,
+          up: $(this).attr("change_up_to")
+        }
+      }
+    });
+
+    request.success(function(data) {
+      var append_to_element = "#review_id-" + data.review_id + " .voting";
+
+      $("#review_id-" + data.review_id + " span").remove();
+
+      if (data.up === 1) {
+        $(append_to_element).append(create_span(0, '+', 'voted' ));
```

Comment:
Now do `vote_container.append(create_span(0, '+', 'voted'));` Repeat for all the lines below as well.